apcd-cms/src/apps/submission_form/templates/submission_form/submission_form.test.html

```diff
@@ -0,0 +1,2965 @@
+      }
+      /* for Contacts */
+      [name^="contact_phone"] {
+        width: 17ch !important; /* to overwrite site.css */
+        box-sizing: content-box;
+      }
+    </style>
+    <style id="form-css-request-submit--radio-fields-layout">
+      /* To make radio fields take up less space */
+      ul:is([id^="types_of_files"], [id^="submit_methods"]) {
+        display: flex;
+        flex-wrap: wrap;
+        column-gap: 1em;
+      }
+
+      /* To align "No (such files)" */
+      ul:is([id^="types_of_files"], [id^="submit_methods"]) {
+        width: 100%;
+      }
+      ul:is([id^="types_of_files"], [id^="submit_methods"])
+        li:has(input[value="true"]) {
+        flex-basis: 20ch;
+      }
+
+      /* To align "No (such files)" */
+      ul:is([id^="types_of_files"], [id^="submit_methods"])
+        li:has(input[value="false"]) {
+        font-style: italic;
+      }
+    </style>
+
+    <style id="form-css-request-submit--hide-unusable-field">
+      /* HACK: FP-1808: To hide field we want to use (until it can be used) */
+      div.field-wrapper.checkboxselectmultiple:not(.required) {
+        display: none;
+      }
+    </style>
+
+    <style id="form-css-request-submit--fp-1808">
+      /* HACK: FP-1808: To warn admin and user that this field is broken */
+      div.field-wrapper.checkboxselectmultiple.required > label::after {
+        display: block;
+        margin-top: 0.5em;
+
+        /* To mimic c-message */
+        /* https://github.com/TACC/tup-ui/blob/main/libs/core-styles/src/lib/_imports/components/c-message.css */
+        padding: 15px 20px;
+        border: var(--global-border--thick);
+        font-size: var(--global-font-size--small);
+      }
+      html[class*="cms-structure-mode-"]
+        div.field-wrapper.checkboxselectmultiple.required
+        > label::after {
+        content: "A multi checkbox field that is required is not well implemented. Do not require this field or replace it with multiple required Yes/No radio fields.";
+
+        /* To mimic c-message c-message-error */
+        /* https://github.com/TACC/tup-ui/blob/main/libs/core-styles/src/lib/_imports/components/c-message.css */
+        color: var(--global-color-danger--dark);
+        background-color: var(--global-color-danger--weak);
+        border-color: var(--global-color-danger--normal);
+      }
+      html:not([class*="cms-structure-mode-"])
+        div.field-wrapper.checkboxselectmultiple.required
+        > label::after {
+        content: "This field is not well implemented. Select all boxes to bypass the error with this field.";
+
+        /* To mimic c-message c-message-warning */
+        /* https://github.com/TACC/tup-ui/blob/main/libs/core-styles/src/lib/_imports/components/c-message.css */
+        color: var(--global-color-warning--dark);
+        background-color: var(--global-color-warning--weak);
+        border-color: var(--global-color-warning--normal);
+      }
+    </style>
+  </aside>
```

Comment:
**Notice.** You may keep these here, for now.

<details>

- They are specific to this page.
- There is no way (yet) for me to add per-page Snippets to a custom app.
- Having the styles committed is better than Snippet anyway.

</details>

**Eventually.** Now (as time allows) or when available (cuz we know the rush), they should at least be moved into a stylesheet in `apcd-cms/static/apcd-cms/css/`.